File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson01DefiningClasses/Exercise/pr03_opinion_poll/Person.java
package Lesson01DefiningClasses.Exercise.pr03_opinion_poll;

public class Person {
    private String name;
    private int age;

    Person(String name, int age) {
        super();
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    int getAge() {
        return age;
    }
}

File: C#-OOP-Basics/Problems Course Instance 2016/Defining-Classes/02.Creating-Constructors/Person.cs
﻿class Person
{
    public string name;
    public int age;

    public Person()
    {
        this.name = "No name";
        this.age = 1;
    }

    public Person(int age)
    {
        this.age = age;
        this.name = "No name";
    }

    public Person(string name, int age)
    {
        this.name = name;
        this.age = age;
    }
}